main.go

```diff
@@ -72,7 +73,9 @@ func initServer(configStorage model.ConfigurationStorage) model.Server {
 		log.Panicln("Cannot init database composer:", err)
 	}
 
-	srv, err := server.NewServer(server.ServerSettings, dbComposer, configStorage, nil)
+	srv, err := server.NewServer(server.ServerSettings, dbComposer, configStorage, &model.CorsOptions{
```

Comment:
Is this makes cors enabled for everyone? I believe the idea was to limit cords to specific URIs only.

Reply:
This cors enabled for API only and I think this is normal that cors enabled by default, because primary goal for API is external call from browser/app. Also there is originChecker that disable origin if it not allowed in redirect urls.

Reply:
yep, make sense now. Thanks for the explanation.